docs/source/developer/interactives.rst

```diff
@@ -161,20 +116,28 @@ Items of interest from this diagram:
   .. code-block:: html
 
     {% extends interactive_mode_template %}
+  
+    {% load i18n %}
+    {% load static %}
 
     {% block html %}
       <!-- The HTML of your interactive goes here -->
     {% endblock html %}
 
     {% block css %}
-      <!-- The CSS of your interactive goes here -->
+      <!-- The CSS of your interactive goes here. Example of how to link to scss file below. -->
+      <link rel="stylesheet" href="{% static 'interactives/<interactive-slug>/scss/<interactive-slug>.scss' %}">
     {% endblock css %}
 
     {% block js %}
-      <!-- The JavaScript of your interactive goes here -->
+      <!-- The JavaScript of your interactive goes here. Example of how to link to js file below. -->
+      <script type="text/javascript" src="{% static 'interactives/<interactive-slug>/js/<interactive-slug>.js' %}"></script>
     {% endblock js %}
 
   The first line ``{% extends interactive_mode_template %}`` is required for all interactives.
+  The second line ``{% load i18n %}`` is needed if you have translatable text in your template.
+  The third line ``{% load static %}`` allows us to serve additional files such as images, JavaScript or CSS.
```

Comment:
Inconsistent use of 'us' and 'you' in these two sentences. Pick one, I don't mind which - but perhaps it'd be best to avoid both